server/routes/v2Manage.ts

```diff
@@ -46,31 +45,27 @@ export default function routes(controllers: Controllers, router: Router, service
     allowedRoles: ['future_manager', 'cru_member'],
     allowedPermissions: ['cas1_out_of_service_bed_create'],
   })
-  get(paths.v2Manage.premises.beds.overbookings.show.pattern, bedsController.overbookings(), {
-    auditEvent: 'SHOW_OVERBOOKINGS',
-    allowedRoles: ['future_manager'],
-  })
 
   // Bookings
   get(paths.v2Manage.bookings.show.pattern, bookingsController.show(), {
     auditEvent: 'SHOW_BOOKING',
-    allowedRoles: ['future_manager'],
+    allowedRoles: ['workflow_manager', 'future_manager'],
```

Comment:
The V1 Manage version was accessible by the role `workflow_manager`, but is now redirected. No user has this role in prod, but we'll remove it completely in a separate PR, so for now we add it to this endpoint to satisfy the tests.